helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -99,7 +99,7 @@ alertmanager:
         # See: https://github.com/helm/helm/issues/2798#issuecomment-467319526
         text: |-
           <!channel>
-          *Cluster:* grafana.{{ .Values.global.opsDomain }}
+          *Cluster:* {{ .Values.user.grafana.subdomain }}.{{ .Values.global.opsDomain }}
```

Comment:
This should be `.Values.prometheus.grafana.subdomain` because we might want to have those two different.

Same for rest of this file